orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -350,13 +469,13 @@ private static RollbackDetails fetchRollbackDetails(
       throw new IllegalStateException(
           format(
               "Found more than one server group with the same name! (serverGroupName: %s)",
```

Comment:
@lorin I know this is not yours, but this error message is misleading. It's not that there are multiple with same name it's that we didn't find a server group to roll back to.
Can you update it please, since you already have this PR out?

Reply:
Fixed in 722ebd4d3627444ac91c40c25f80b4de48f01daa